test/helpers/reducers.js

```diff
@@ -1,10 +1,16 @@
 import { ADD_TODO } from './actionTypes';
 
+
+var id = (state = []) =>
```

Comment:
Let's change this to be  `function nextId(state = [])`.
Also please run `npm run lint` to fix the errors.

Thanks!